chapters/extend/driver.rst

```diff
@@ -1,16 +1,153 @@
-Guideline When Adding New Driver
-================================
+Development Guide: Add new Drivers
+====================================
 
-You come to this page and you realize the existing executor/driver logic may not suit you need, so you want to write a new driver and add to Jina.
+.. meta::
+   :description: Development Guide: Add new Drivers
+   :keywords: Jina, driver
 
-Or *do* you?
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
```

Comment:
I actually think we should replace this with 
`.. warning::
    This is an advanced developer guide. The existing drivers should cover most use cases. If you add custom drivers, the Jina community may not be able to provide you with development support. `

Reply:
I'm open to suggestions about the wording, but I want to make it clear that is not common. 

@bwanglzu @JoanFM what do you think?

Reply:
I like it @FionnD